frontend/src/global_styles/layout/_main_menu.sass

```diff
@@ -226,7 +226,8 @@ a.main-menu--parent-node
         display: list-item
         .main-menu--children-menu-header
           display: none
-
+        .spot-icon_enterprise-badge
+          margin-left: 0.25rem
```

Comment:
The icon spacing does not look exactly like intended on two places:
* http://localhost:3000/users, Placeholder user
* http://localhost:3000/admin/settings/work_packages, Custom actions

In fact the spacing is due to a white space character, and not the intended 0.25em margin. It's because the menu root has a `closed` style, and thus css selector `open` can't apply.

Maybe move the rule one level up, like at the `menu_root` level instead?